CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## [15.5.3] - 2025-XX-XX
+* Added support for user CA certificates
+* Fixed issue where many Go server implementations of WebDAV did not work with the WebDAV client in Seedvault
+* Bumped the app data quota from 1GB to 3GB
+* Improved handling of metered networks, if disallowed, the backup process will be aborted
+* Fixed backup errors with USB when file and app backup are both on
+* Fixed overdue backups not automatically starting when USB drives are plugged in
+* The size of each app backup is now shown on the restore screen
+* Fixed a common error (StaleDataException) causing backups to fail
+* Fixed error message when no backups are available to restore
+* Implemented a wrapper for the backend with a retrying mechanism, giving us less common errors
+* Updated dependencies for 15 QPR1
+* Merged translations
```

Comment:
Added this since this should be straight forward enough?

Reply:
Not sure, we are ready to merge this now, but translations haven't been merged.

Reply:
```
git fetch https://hosted.weblate.org/git/calyxos/seedvault/ android14 && git cherry-pick FETCH_HEAD
```

should work and you don't rely on unauthenticated #836

Reply:
That's ok, I've removed that in the last push

Reply:
Thanks!

Reply:
I thought you guys would just merge that in, hence added that, feels a little bit waiting for (?) given this should be relatively quick and keeping these things waiting can cause other issues (fuzzy translations in case of strings changing), I suppose @chirayudesai is the one with control to Weblate itself though